torchci/components/metrics/panels/TimeSeriesPanel.tsx

```diff
@@ -148,9 +164,12 @@ export default function TimeSeriesPanel({
     yAxisFieldName
   );
 
+  // Add extra padding when the legend is active
+  const legend_padding = groupByFieldName !== undefined ? 200 : 48;
```

Comment:
nit: I feel like having a legend would still be helpful even if there's only 1 series.

Reply:
What value would you expect the legend to have? When there's no group by, there's no logical value to assign to the legend that you don't already get from e.g. the chart title.  

If there's a value you'd find useful perhaps we can add that to the title or either the X or Y axis labels.